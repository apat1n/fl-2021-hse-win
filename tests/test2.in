eval X Y = X || Y;
main = eval true false;