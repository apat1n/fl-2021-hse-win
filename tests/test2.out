TREE
Node 0: VARS
No edges
Node 1: Y
No edges
Node 2: VARS
Edges to [1, 0]
Node 3: X
No edges
Node 4: VARS
Edges to [3, 2]
Node 5: eval
No edges
Node 6: EXP
Edges to [5, 4]
Node 7: X
No edges
Node 8: EXP
Edges to [7]
Node 9: Y
No edges
Node 10: EXP
Edges to [9]
Node 11: OR
No edges
Node 12: GOAL
Edges to [8, 11, 10]
Node 13: VARS
No edges
Node 14: false
No edges
Node 15: VARS
Edges to [14, 13]
Node 16: true
No edges
Node 17: VARS
Edges to [16, 15]
Node 18: eval
No edges
Node 19: EXP
Edges to [18, 17]
Node 20: EQ
No edges
Node 21: SC
No edges
Node 22: MAIN
Edges to [20, 19, 21]
Node 23: EQ
No edges
Node 24: SC
No edges
Node 25: START
Edges to [6, 23, 12, 24, 22]
